### ISSUE TITLE
licenseFile becomes README for almost all modules

### ISSUE THREAD

**Issue by ibeucaly** [davglass/license-checker#112]
The file named README.* is treated as `licenseFile` for almost all modules since v12.0.0.

For example, output  with `license-checker@12.0.0` is like below:
```
'license-checker@12.0.0':
   { licenses: 'BSD-3-Clause',
     repository: 'https://github.com/davglass/license-checker',
     publisher: 'Dav Glass',
     email: 'davglass@gmail.com',
     licenseFile: '/Users/nemuri/works/github/ibeucaly/license-checker/README.md' },
```
while with `license-checker@11.0.0`:
```
'license-checker@11.0.0':
   { licenses: 'BSD-3-Clause',
     repository: 'https://github.com/davglass/license-checker',
     publisher: 'Dav Glass',
     email: 'davglass@gmail.com',
     licenseFile: '/Users/nemuri/works/github/ibeucaly/license-checker/LICENSE' },
```

I think the README file should be treated as the license file when no other license file exists.

**Comment by klimashkin** [comment]
The same issue, `licenseChecker.init` for all 187 dependencies returns README.md files since version `12`

**Comment by klimashkin** [comment]
The problem is [here](https://github.com/davglass/license-checker/blob/a2295073d0178dca1b88e85ea8f52c21d35e52fc/lib/index.js#L149), `fs.readdirSync` doesn't guarantee order.
Moreover returning README should be an option, otherwise I can't recognize if license-checker can't find license, because allmost all modules contain README, but license - don't.

cc @davglass @helio-frota

**Comment by davglass** [comment]
Published a fix in `license-checker@13.0.1`
